modules/cnf-topology-aware-lifecycle-manager-about-cgu-crs.adoc

```diff
@@ -33,218 +45,373 @@ After {cgu-operator} completes a cluster update, the cluster does not update aga
 * The `clusters` field specifies a list of clusters to update.
 * The `canaries` field specifies the clusters for canary updates.
 * The `maxConcurrency` field specifies the number of clusters to update in a batch.
 
-You can use the `clusters` and the `clusterSelector` fields together to create a combined list of clusters.
+You can use the `clusters`, `clusterLabelSelector`, and `clusterSelector` fields together to create a combined list of clusters.
 
 The remediation plan starts with the clusters listed in the `canaries` field. Each canary cluster forms a single-cluster batch.
 
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1alpha1
+kind: ClusterGroupUpgrade
+metadata:
+  creationTimestamp: '2022-11-18T16:27:15Z'
+  finalizers:
+    - ran.openshift.io/cleanup-finalizer
+  generation: 1
+  name: talm-cgu
+  namespace: talm-namespace
+  resourceVersion: '40451823'
+  uid: cca245a5-4bca-45fa-89c0-aa6af81a596c
+Spec:
+  actions:
+    afterCompletion:
+      deleteObjects: true
+    beforeEnable: {}
+  backup: false
+  clusters: <1>
+    - spoke1
+  enable: true <2>
+  managedPolicies: <3>
+    - talm-policy
+  preCaching: false
+  remediationStrategy: <4>
+    canaries: <5>
+        - spoke1
+    maxConcurrency: 2 <6>
+    timeout: 240
+  clusterLabelSelectors: <7>
+    - matchExpressions:
+      - key: label1
+      operator: In
+      values:
+        - value1a
+        - value1b
+  batchTimeoutAction: <8>
+status: <9>
+    clusters:
+      - name: spoke1
+        state: complete
+      - currentPolicy:
+          name: talm-policy
+          status: NonCompliant
+        name: spoke2
+        state: timedout
+    computedMaxConcurrency: 2
+    conditions:
+      - lastTransitionTime: '2022-11-18T16:27:15Z'
+        message: All selected clusters are valid
+        reason: ClusterSelectionCompleted
+        status: 'True'
+        type: ClustersSelected <10>
+      - lastTransitionTime: '2022-11-18T16:27:15Z'
+        message: Completed validation
+        reason: ValidationCompleted
+        status: 'True'
+        type: Validated <11>
+      - lastTransitionTime: '2022-11-18T16:37:16Z'
+        message: The cluster backup is in progress
+        reason: NotStarted
+        status: 'False'
+        type: Progressing
+      - lastTransitionTime: '2022-11-18T16:37:16Z'
+        message: Policy remediation took too long
+        reason: TimedOut
+        status: 'False'
+        type: Succeeded
+    managedPoliciesForUpgrade:
+      - name: talm-policy
+        namespace: talm-namespace
+    managedPoliciesNs:
+      talm-policy: talm-namespace
+    remediationPlan:
+      - - spoke1
+        - spoke2
+    status:
+      currentBatch: 1
+      currentBatchRemediationProgress:
+        spoke1:
+          state: Completed
+        spoke2:
+          policyIndex: 0
+          state: InProgress
```

Comment:
These lines should be removed if we are showing a succeeded or timeout CGU

Reply:
ok